### ISSUE TITLE
Cannot read property 'getCache' of undefined

### ISSUE THREAD

**Issue by spicq-rambobafet** [baptooo/safely-set-inner-html#5]
Hi,
When i'm trying to instantiate `mySafelySetInnerHTML` just like in your example, i get this error:
> Uncaught (in promise) TypeError: Cannot read property 'getCache' of undefined
    at transform (main-react.js:81988)
    at BannerTop._this.render (main-react.js:71583)
    at finishClassComponent (main-react.js:23463)
    at updateClassComponent (main-react.js:23440)
    at beginWork (main-react.js:23819)
    at performUnitOfWork (main-react.js:25787)
    at workLoop (main-react.js:25896)
    at HTMLUnknownElement.callCallback (main-react.js:14513)
    at Object.invokeGuardedCallbackDev (main-react.js:14552)
    at invokeGuardedCallback (main-react.js:14409)

I have a file safelySetInnerHTML.js, copy/past of the configuration example, that i call in my component like this: 
```javascript
import mySafelySetInnerHTML from "[path to the file]/safelySetInnerHTML";
...
class MyComponent extends Component {
    render = () => {
        return (
            <span>
                {mySafelySetInnerHTML(text)}
             </span>
        );
    }
}
```

But if I create a new instance, just like in the Usage section, without defining a custom configuration it works fine.
```javascript
import SafelySetInnerHtml from 'safely-set-inner-html';
...
const instance = new SafelySetInnerHtml();
...
<span>instance.transform(text)}</span>
```

Any idea of what I'm missing ? :)

**Comment by baptooo** [comment]
Oh right you can't export the function **transform** as mentionned in the example because it is called oustide the instance 😞 

Maybe I will bind automatically the methods to the instance in its constructor.

Meanwhile, you can just transform your configuration file temporary like this:

```diff
// mySafelySetInnerHTML.js
import SafelySetInnerHTML from 'safely-set-inner-html';

const mySafelySetInnerHTML = new SafelySetInnerHTML({
  ALLOWED_TAGS: [
    'a',
    'strong'
  ],
  ALLOWED_ATTRIBUTES: [
    'href',
    'class'
  ]
});

-export default mySafelySetInnerHTML.transform;
+export default mySafelySetInnerHTML.transform.bind(mySafelySetInnerHTML);
```

**Comment by spicq-rambobafet** [comment]
Ok I will try that.
Thanks ❤️

**Comment by baptooo** [comment]
@Rambobafet please find the fix in the [latest release (v0.0.7)](https://github.com/baptooo/safely-set-inner-html/releases/tag/v0.0.7)
